faq.md

```diff
@@ -58,6 +69,11 @@ stock platform apps can also be built with Flutter.
 You can build full-featured apps with Flutter, including
 camera, geolocation, network, storage, 3rd-party SDKs, and more.
 
+### Who makes Flutter?
+
+Flutter is an open source project, with contributions from Google
+and the community.
```

Comment:
we are part of the community, this makes it sound like we're separate.
Maybe "Flutter is an open source project, primarily led by Google engineers, with many contributions from other members of the wider community."?

Reply:
works for me, thanks!